docs/get_started.rst

```diff
@@ -1,17 +1,369 @@
+| Minimum code coverage percentage                                 | Sets the minimum allowed code coverage percentage for your project.                                                                                                                                                                                                                                   |
++------------------------------------------------------------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+
+
+Verify Project Initialization
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Verify project initialization was successful by verifying that ``cumulusci.yml`` exists and has contents.
+
+.. code-block:: console
+
+    $ cat cumulusci.yml
+    project:
+        name: SampleProjectName
+        package:
+            name: SamplePackageName
+            namespace: sampleNamespace
+        .
+        .
+        .
+
+The ``cumulusci.yml`` file configures your project-specific tasks, flows, and customizations.
+
+For more information regarding configuration, check out our `project configuration <#TODO internal ref here>`_ section of the docs.
+You can even add and commit it to your Git repository:
+
+.. code-block:: console
+
+    $ git add cumulusci.yml
+    $ git commit -m "Initialized CumulusCI Configuration"
+
+
+
+Add Your Repo to GitHub
+^^^^^^^^^^^^^^^^^^^^^^^
+With your ``cumulusci.yml`` file committed, create a repository on GitHub for your new project and push your changes there.
+There are multiple ways to do this:
+
+* Our `Community Project Development with CumulusCI <https://trailhead.salesforce.com/content/learn/modules/community-project-development-with-cumulusci?trail_id=build-applications-with-cumulusci>`_ module covers GitHub Desktop in the "Set Up the Project" section.
+* If you prefer the command line, GitHub has excellent guides on both `git remote <https://github.com/git-guides/git-remote>`_ and `git push <https://github.com/git-guides/git-push>`_.
+
+
+Convert an Existing Salesforce Project
+--------------------------------------
+Converting an existing Salesforce project to use CumulusCI may follow a number of different paths, depending on whether you're practicing the Org Development Model or the Package Development Model; whether or not you're already developing in scratch orgs; and the complexity of your project's dependencies on the org environment.
+
+If you're developing in scratch orgs, then you likely only need to do `project setup`_ and `org shape setup`_.
+
+If you're developing in persistent orgs, then you likely want to go through **all** of the following sections. Your experience may vary. 
```

Comment:
The phrasing of this irks me. Is there something we can/should add to it?

Reply:
```suggestion
If you're developing in scratch orgs, then you likely only need to do `project setup`_ and `org shape setup`_.

If you're developing in persistent orgs and not already using source control, you'll also need to complete `metadata capture`_. Note that adopting source control and moving from persistent orgs into scratch orgs are case-specific processes; your experience may vary and you may need to undertake additional work as part of those transitions.  
```

Reply:
@davidmreed & @davisagli: Do we need to change the language for "metadata capture" here as well? Should we say "retrieving metadata"?

Reply:
Yeah, I think we can say "you'll also need to retrieve the existing metadata from the org." (or rephrase if that's better for linking). Anywhere we said "capture" is out of a habit we're trying to break, not intentional.

Reply:
Agreed